src/main/java/seedu/duke/todo/Event.java

```diff
@@ -0,0 +1,44 @@
+package main.java.seedu.duke.todo;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event happening at a particular time.
+ */
+public class Event extends Task {
+    protected LocalDate at;
```

Comment:
Should the variable name be changed into a more meaningful one like "time" or "date"?